README.md

```diff
@@ -0,0 +1,28 @@
+# flapjak
+
+Featherweight LDAP server configured with jsonnet and Kubernetes.
+
+I need a little LDAP server. I don't have many records to serve, but some record
+types I want to serve are `automountMap` and `automount`. None of the little
+LDAP servers I can find serve these records. So I run OpenLDAP in order to serve
+these. It is big and complicated and annoying to restore from backup for my
+situation.
+
+So, using [gldap], it seems I should be able to serve any sort of record I want.
```

Comment:
Reading this it would have been nice to have a half sentence what gldap is about rather than having to follow the link. </my-opinion>